From a47c5838e9f445ab887ad927706b11ccbb181364 Mon Sep 17 00:00:00 2001
From: David Lamparter <equinox@opensourcerouting.org>
Date: Thu, 21 Jun 2012 09:55:38 +0200
Subject: [PATCH] isisd: fix typo in topology generator (BZ#731)

There was a "lsp->" missing before "level" in line 2416.
(introduced by git commit e38e0df)

Reported-by: Seblu <seblu@seblu.net>
Signed-off-by: David Lamparter <equinox@opensourcerouting.org>
---
 isisd/isis_lsp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/isisd/isis_lsp.c b/isisd/isis_lsp.c
index 5c1e993..082e9dc 100644
--- a/isisd/isis_lsp.c
+++ b/isisd/isis_lsp.c
@@ -2413,7 +2413,7 @@ top_lsp_refresh (struct thread *thread)
   isis_dynhn_insert (lsp->lsp_header->lsp_id, lsp->tlv_data.hostname,
 		     IS_LEVEL_1);
 
-  lsp->lsp_header->lsp_bits = lsp_bits_generate (level,
+  lsp->lsp_header->lsp_bits = lsp_bits_generate (lsp->level,
                                                  lsp->area->overload_bit);
   rem_lifetime = lsp_rem_lifetime (lsp->area, IS_LEVEL_1);
   lsp->lsp_header->rem_lifetime = htons (rem_lifetime);
-- 
1.7.11

